src/main/java/fi/aalto/cs/apluscourses/intellij/model/SettingsImporterImpl.java

```diff
@@ -91,11 +104,54 @@ public void importProjectSettings(@NotNull Project project, @NotNull Course cour
 
     for (String fileName : fileNames) {
       Path path = Paths.get(fileName);
+      // The ZIP contains a .idea directory with all of the settings files. We want to extract the
+      // files to the .idea directory without the .idea prefix, as otherwise we would end up with
+      // .idea/.idea/<settings_files>.
       Path pathWithoutRoot = path.subpath(1, path.getNameCount());
-      zipFile.extractFile(path.toString(), settingsPath.toString(), pathWithoutRoot.toString());
+      zipFile.extractFile(path.toString(), target.toString(), pathWithoutRoot.toString());
     }
+  }
+
+  private static final String WORKSPACE_XML_COMPONENT_NAME = "CompilerWorkspaceConfiguration";
+  private static final String WORKSPACE_XML_OPTION_NAME = "AUTO_SHOW_ERRORS_IN_EDITOR";
+  private static final String WORKSPACE_XML_OPTION_VALUE = "false";
```

Comment:
not a big deal, but I'd suggest moving the declaration to the top of the file, to follow the convention